back/src/companies/typeDefs/company.objects.graphql

```diff
@@ -707,3 +732,20 @@ type SignatureAutomation {
   "Date de création de l'automatisation de signature"
   createdAt: DateTime!
 }
+
+"""
+Informations sur un transfert administratif entre deux entreprises
+"""
+type AdministrativeTransfer {
+  id: ID!
+  "Entreprise qui initie le transfert"
+  from: CompanyPrivate!
+  "Entreprise qui reçoit le transfert"
+  to: CompanyPrivate!
+  "Date de création du transfert"
+  createdAt: DateTime!
+  "Date de validation du transfert"
+  approvedAt: DateTime
+  "Statut du transfert"
+  status: AdministrativeTransferStatus
+}
```

Comment:
Si je comprends bien la mutation createAdministrativeTransfer, on peut faire un transfert à une company dont on est pas membre/admin. Donc il y a un scénario:
- je crée un compte trackdechets avec une entreprise
- je la mets en sommeil direct
- je crée un transfert avec une entreprise
- je peux récup ses infos privées (CompanyPrivate donc securityCode, liste des membres,...)
peut-être utiliser CompanyPublic dans "to"?

Reply:
Bien vu ! Faut en effet passer sur du CompanyPublic 👍